mlflow/tracking/_tracking_service/client.py

```diff
@@ -873,6 +874,22 @@ def download_artifacts(self, run_id, path, dst_path=None):
         """
         return self._get_artifact_repo(run_id).download_artifacts(path, dst_path)
 
+    def _log_url(self, run_id):
+        if not isinstance(self.store, RestStore):
+            return
+        host_url = self.store.get_host_creds().host
+        run_info = self.store.get_run(run_id).info
+        experiment_id = run_info.experiment_id
+        run_name = run_info.run_name
+        if self.tracking_uri == "databricks":
+            experment_url = f"{host_url}/ml/experiments/{experiment_id}"
+        else:
+            experment_url = f"{host_url}/#/experiments/{experiment_id}"
+        run_url = f"{experment_url}/runs/{run_id}"
+
+        _logger.info(f"🏃 View run {run_name} at: {run_url}.")
+        _logger.info(f"🧪 View experiment at: {experment_url}.")
```

Comment:
Do we need experiment? If there're lots of runs (and normally people just use one experiment) then this displays a lot

Reply:
Good question! Mostly aligning with wandb, also experiment view is much preferred by Mosaic research team because they generally use the chart view to compare across runs.